packages/connect-form/src/advanced-options-tabs/advanced-options-tabs.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { Tabs, Tab } from '@leafygreen-ui/tabs';
+
+import GeneralTab from './general-tab';
+import SSLTab from './ssl-tab';
+import SSHTunnelTab from './ssh-tunnel-tab';
+import AdvancedTab from './advanced-tab';
+
+interface TabObject {
+  name: string,
+  component: React.FunctionComponent
+}
+
+function AdvancedOptionsTabs(): React.ReactElement {
+  
+  function buildTab(tabObject: TabObject, idx: number): React.ReactElement {
```

Comment:
Super random comment: normally we would call this method `renderTab`.

In this specific case, since this is basically already another functional component we could move it outside the function and also just be: `function AdvancedOptionTab(tabObject, TabObject, idx: number)`